.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Description
```

Comment:
consider also adding here a section on the top like in the bug report. So `name`, `about` and `labels`